### DISCUSSION TITLE
Issues receiving emails sent with default `sendmail`

### DISCUSSION THREAD

**Discussion by shot-codes** [pocketbase/pocketbase#1193] [Q&A]
Perhaps I shouldn't expect it to just work, but the admin UI is telling me emails are sent successfully when I test them. However I do not receive any emails. I'm running MacOS

**Comment by ganigeorgiev** [comment]
The "sent successfully" message means that `sendmail` exited with a success code, but that doesn't necessarily mean that the email was successfully sent since it could be rejected at a later stage after some delay from the receiver (this is often the case with gmail for example).

On linux usually you can inspect your sendmail logs with `cat /var/spool/mail/youruser` but I'm not sure if this is the same for MacOS.

For testing purposes, I'd recommend using a fake smtp service, eg. [mailtrap.io](https://mailtrap.io/) (they have free plan), since it is guaranteed that you'll be able to send and preview the email.
